test_unstructured/partition/docx/test_docx.py

```diff
@@ -866,3 +866,18 @@ def mock_document_file_path(mock_document: Document, tmp_path: pathlib.Path) ->
     filename = str(tmp_path / "mock_document.docx")
     mock_document.save(filename)
     return filename
+
+
+def test_ids_are_unique_and_deterministic():
+    elements = partition_docx("example-docs/duplicate-paragraphs.docx")
+
+    ids = [e.id for e in elements]
+    assert ids == [
+        "2f22d82eea1faf5f40dac60cef52700e",
+        "ca9e1f448e531a5152d960e14eefc360",
+        "9ddeacb172ac17fb45e6f3f15f3c703d",
+        "a4fd85d3f4141acae38c8f9c936ed2f3",
+        "44ebaaf66640719c918246d4ccba1c45",
+        "f36e8ebcb3b6a051940a168fe73cbc44",
+        "532b395177652c7d61e1e4d855f1dc1d",
```

Comment:
don't we just care about the length of distinct element id's rather than the id's themselves? (not a blocker)

Reply:
At some point I've been doing this in combination with:
```
assert len(ids) == len(set(ids))
```
Steve suggested that it's redundant since checking hashes explicitly already ensures their uniqueness.